tomcat9/src/main/java/com/linecorp/armeria/internal/server/tomcat/LogWrapper.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.server.tomcat;
+
+import org.apache.juli.logging.Log;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Wraps an existing {@link Logger}.
+ */
+public final class LogWrapper implements Log {
```

Comment:
How about moving into `c.l.a.server.tomcat` and make this package private?

Reply:
I'm not sure if relying on external packages is a good way to go. 🤔

Reply:
I think if a class can be hidden from `public`, it would be better than `public` in `internal` package.

Reply:
Oh! I misunderstood at first. 🤣 I'll fix this.